scipy/stats/_continuous_distns.py

```diff
@@ -7040,11 +7040,25 @@ def _stats(self, df):
     def _entropy(self, df):
         if df == np.inf:
             return norm._entropy()
+
         half = df/2
         half1 = (df + 1)/2
-        return (half1*(sc.digamma(half1) - sc.digamma(half))
-                + np.log(np.sqrt(df)*sc.beta(half, 0.5)))
 
+        # _lazywhere complains if the functions don't have inputs
+        def regular(df):
+            return (half1*(sc.digamma(half1) - sc.digamma(half))
+                    + np.log(np.sqrt(df)*sc.beta(half, 0.5)))
+
+        def asymptotic(df):
+            # gammaln(x) ~ x * ln(x) - x - 0.5 * ln(x) + 0.5 * ln(2 * pi)
+            # psi(x) ~ ln(x) - 1 / (2 * x)
+            # B(x, y) ~ gamma(x) * y ** (-x)
+            # (x / 2) * (psi(x + 1) - psi(x)) ~ 0.5
+            h = 0.5 + 0.5 * np.log(np.pi) + 0.5 * np.log(2) + 1 / df
```

Comment:
```suggestion
            h = norm._entropy() + 1 / df
```
The entropy of the normal distribution is: $\frac{1}{2}(1+\ln(2\pi))$.
This makes the formula easier to understand (especially with the special case of `inf` in the beginning of the function). I am thinking of adjusting this in my related PRs as well. For the Nakagami distribution this was also done (see [here](https://github.com/scipy/scipy/blob/2fc01bebee4a2846282067835f704fe1802a2c0e/scipy/stats/_continuous_distns.py#L6746)).

Reply:
Done. Interesting that this is the second distribution we've looked at where the asymptotic expansion was the normal entropy.

Reply:
Yep, that is indeed interesting. For infinite degrees of freedom, the t distribution becomes a normal distribution. For the various gamma distribution, it probably arises from the central limit theorem.